frontend/public/components/utils/resource-log.jsx

```diff
@@ -17,14 +17,22 @@ const streamStatusMessages = {
   streaming: 'Log streaming...'
 };
 
+export const logStatuses = {
+  eof: 'eof',
+  loading: 'loading',
+  paused: 'paused',
+  streaming: 'streaming'
+};
```

Comment:
I guess I don't see how this is better since you can typo `logStatuses.streming` just as easily as the string `"streming"`?

`enum` would be different because the compiler would catch a typo, but that's not possible here

Reply:
It's also possible to accidentally reassign the value of `logStatuses.streaming`, which would result in a really hard to detect bug. That's not possible with string literals. So I'd argue this is more error prone than what we had.